src/libvirtApi/common.js

```diff
@@ -206,8 +210,20 @@ export async function getLibvirtVersion({ connectionName }) {
     store.dispatch(updateLibvirtVersion({ libvirtVersion: version.v }));
 }
 
+async function getCapabilities({ connectionName }) {
+    try {
+        const [capabilitiesXML] = await call(connectionName, "/org/libvirt/QEMU", "org.libvirt.Connect", "GetCapabilities", [],
+                                             { timeout, type: "" });
+        const capabilities = parseDumpxmlForCapabilities(capabilitiesXML);
+        store.dispatch(setCapabilities({ capabilities }));
+    } catch (ex) {
+        console.warn("NodeGetMemoryStats failed:", ex.toString());
+        throw ex;
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1280-20240129-120008-4a545550-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)